README.md

```diff
@@ -1,2 +1,45 @@
 # Hello-Microverse
-This is a trial code review repo
+This is a trial code review repo which is about showing Hello Microverse as a header.
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+- Webhint and Stylelint
+
+## Live Demo
```

Comment:
### [OPTIONAL]

- [ ] It would be great to have a live link for your project. This will make your project even more professional. Kindly you can use [this site](https://raw.githack.com/) to create a live link for your project.